scratchpad/components/todo/controllers/TodoController.ts

```diff
@@ -0,0 +1,23 @@
+import {authenticate, authenticatedUser} from '@loopback/authentication';
+import {role} from '@loopback/authorization';
+import {todoRepo} from '../bindings/TodoRepo.ts';
+import {get} from '@loopback/core';
+
+export class TodoController {
```

Comment:
Do we automatically bind controllers? Otherwise, the bindings have to deal with it.